core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```diff
@@ -345,7 +346,7 @@ public void append(FieldStats stats) {
         }
 
         @Override
-        protected java.lang.Double valueOf(String value) {
+        protected java.lang.Double valueOf(String value, String optionalFormat) {
```

Comment:
I'll add the unsupported operation exception.

there is jdoc on the abstract method. I'll add there it is up to the implementation if the format parameter is used.